editoast/src/views/timetable/similar_trains.rs

```diff
@@ -197,16 +178,12 @@ pub(in crate::views) async fn similar_trains(
     State(AppState {
         db_pool,
         speed_limit_tag_ids,
-        valkey_client,
-        core_client,
-        config,
+        trains_traffic,
```

Comment:
Should we return an error when the traffic is empty? That could be useful when we forget to provide the traffic file, instead of just a "no similar train" response.

Reply:
I set the error as a 404 (no trains traffic), and the front handles this case.